misc-scripts/xref_mapping/XrefParser/EntrezGeneParser.pm

```diff
@@ -50,74 +50,27 @@ sub run {
   
 
   my $eg_io = $self->get_filehandle($file);
-  if ( !defined $eg_io ) {
-    print STDERR "ERROR: Could not open $file\n";
-    return 1;    # 1 is an error
-  }
+  croak "Could not open $file\n" unless defined $eg_io;
 
-  my %seen;
+  my $input_file = Text::CSV->new({ sep_char => "\t", empty_is_undef => 1, allow_loose_quotes => 1 }) 
```

Comment:
I would split this into multiple lines, both for better readability and to keep line length under control.